pkg/block/block.go

```diff
@@ -25,6 +25,8 @@ const (
 	MetaFilename = "meta.json"
 	// IndexFilename is the known index file for block index.
 	IndexFilename = "index"
+	// IndexCacheFilename is the canonical name for index cache files.
```

Comment:
Nice, but let's mayb extend this a bit:
```suggestion
	// IndexCacheFilename is the canonical name for index cache file that stores essential inforrmation needed
	// to lookup postings and series' chunk later on. 
```